src/Action/Task/FailureReasons.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace AppBundle\Action\Task;
+
+use AppBundle\Entity\Store;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Yaml\Parser as YamlParser;
+use Symfony\Component\Yaml\Yaml;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class FailureReasons
+{
+
+    public function __construct(
+        private EntityManagerInterface $em,
+        private TranslatorInterface $translator
+    )
+    { }
+
+    private function getDefaultReasons(): array
+    {
+        $path = realpath(__DIR__ . '/../../Resources/config/failure_reasons.yml');
```

Comment:
Could be cached to improve performance

Reply:
If a package is reported as damaged, a photo must be attached to prove the damage.
Generally speaking, it might be interesting (as a dispatch) to be able to configure the situations in which the courier can be forced to attach a photo to prove the event encountered (absent customer -> photo of the front door to prove that he has shown up, for example).
See what urbantz propose for that kind of situations
<img width="216" alt="image" src="https://github.com/coopcycle/coopcycle-web/assets/44605344/50d9bcd5-e3de-40e3-9e49-2a7141753a1f">